app/live_display.py

```
@@ -0,0 +1,106 @@
+from rich.live import Live
+from rich.table import Table
+from rich.console import Console
+from typing import Dict, Optional
+import threading
+from datetime import datetime
+import queue
+
+class ProgressTracker:
+    def __init__(self):
```

Ruff (0.14.7)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)